gradle-plugins/src/main/kotlin/io.opentelemetry.instrumentation.muzzle-check.gradle.kts

```diff
@@ -38,14 +41,37 @@ val muzzleTooling: Configuration by configurations.creating {
   isCanBeConsumed = false
   isCanBeResolved = true
 }
+
 val muzzleBootstrap: Configuration by configurations.creating {
   isCanBeConsumed = false
   isCanBeResolved = true
 }
 
+val muzzleShadow by tasks.registering(ShadowJar::class) {
```

Comment:
I don't understand why this shadow jar is needed?

Reply:
this shades the javaagent module classes themselves

Reply:
Rename it to `shadedModule` then? Or something similar. We don't shade muzzle here, do we?